### ISSUE TITLE
the number of validBarcodes are inconsistent with the cell number of ArchRproj

### ISSUE THREAD

**Issue by Sophiesze** [GreenleafLab/ArchR#314]
Hi,

I was trying to run my 10Xdata. 

I noticed the validBarcodes() parameter in the createArrowFiles() function. I try to input the validbarcode, a total of 17353 cells.

`wc -l output0916-demo/HC-1_validBarcode.csv
11399 output0916-demo/HC-1_validBarcode.csv`

`wc -l output0916-demo/HC-2_validBarcode.csv
5956 output0916-demo/HC-2_validBarcode.csv`

And this is my logFile when creating arrowfiles.

<img width="547" alt="截屏2020-09-16 下午4 15 11" src="https://user-images.githubusercontent.com/53801318/93310792-cd54d500-f837-11ea-9863-9417f8948b93.png">

I thought it would only have 17353 cells in my ArchRproj, but it seems like I got 19276 cells.

<img width="569" alt="截屏2020-09-16 下午4 18 03" src="https://user-images.githubusercontent.com/53801318/93311153-35a3b680-f838-11ea-92d4-7cc420f0d4a7.png">

And I was so confused. How can I make the cell number consistent with the number of validBardcodes?

Thank you so much!

I'm looking forward to your reply.

**Comment by Sophiesze** [comment]
I also tried to input the combined 10X obj (merged using cellranger-atac- aggr with no normalization). The number of validbarcodes is still 17353. However, the cell number of the ArchRproj is 15856 at this time.
<img width="593" alt="截屏2020-09-17 下午2 40 41" src="https://user-images.githubusercontent.com/53801318/93429451-c5f30180-f8f3-11ea-88b5-fd5dc5289290.png">

And then I noticed that 15856 is the intersect barcodes of 19276 and 17353 (the intersect of the barcodes from origin ArchRproj and validBarcodes from 10X). 

Has anyone noticed these issues before? Will it be ok for me to use the 15856 cells for downstream analysis? It really confused me. 

Thanks!
